optimize/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.camunda</groupId>
     <artifactId>zeebe-parent</artifactId>
-    <version>8.8.0-SNAPSHOT</version>
+    <version>8.8.1-SNAPSHOT</version>
```

Comment:
does this mean that from now on, `parent.version` in this file must always match the version from `pom.xml` for zeebe-parent?

does it also mean Optimize and Monorepo must always be built in-sync, one after another for the 8.8.x patches (i.e. if 8.8.1 built for Monorepo, Optimize 8.8.1 also must be built - cannot be skipped, be another patch version etc.)?